src/injectHighlightWrappers.ts

```diff
@@ -203,6 +203,10 @@ function highlightRange(range: Range, wrapper: HTMLElement) {
     }
     if (goDeeper && node.hasChildNodes()) {
       node = node.firstChild as HTMLElement;
+    } else if (!goDeeper && node.contains(endContainer)) {
+      // stops traversing of tree if endContainer is a descendent of current allowed node
+      // this prevents an error that blocks display of highlights ending on an iframe in firefox
+      done = true;
```

Comment:
@TomWoodward two notes: 
- starting highlights on iframes was already working—I knew and then forgot this
- I don't think we need to modify `refineRangeBoundaries` since it doesn't appear to be overlooking any cases or causing errors

Reply:
also this looks good on Chrome and Safari as well

Reply:
> I knew and then forgot this

story of my life